docs/programme/roles/game-design-coord.md

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
We merged this role with head judge, is it worth rewording this page?

Reply:
Probably, though that's a job for later I think.